site/content/docs/5.2/forms/overview.md

```diff
@@ -140,6 +148,13 @@ Use the `.is-disabled` class on `label` to make it appear lighter too.
 
 Use the `.is-required` class on `label` to add a `*` symbol after it.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+The browser generated appearance of the `<select>` dropdown menu does not respect the Orange Design System and cannot be changed due to browser limitations.
```

Comment:
Same as one previous comment, bit confusing here

Reply:
Yes. To be discussed withe the design. This message shall be changed consequently.

Reply:
Delete callout